Cargo.toml

```diff
@@ -34,6 +34,7 @@ std = []
 [dependencies]
 derive_more = "0.99.17"
 serde = { version = "1.0", optional = true, features = ["derive"] }
+regex = "1.10.2"
```

Comment:
Can we remove this line, since it is in build-deps?

Reply:
Sure